feat(docs): update references to removed appengine and artifacts feature flags (#1806)
spinnaker/spinnaker.github.io: reference/artifacts-with-artifactsrewrite/index.md

Keep in mind that the artifact in Spinnaker is a _reference_ to an external resource--it is not the resource itself. The resource itself could be of any type supported by Spinnaker; the artifact is the named JSON object that contains information about the resource.

## Enabling artifact support

If using a version of Spinnaker prior to 1.20, enable support for the standard artifacts UI:

```bash
hal config features edit --artifacts-rewrite true
```

> If using a version of Spinnaker greater than or equal to 1.20, support for the standard artifacts UI is enabled by default.

Reviewer: ```suggestion
If using Spinnaker 1.20 or later, support for the standard artifacts UI is enabled by default.
```
